packages/browser-destinations/src/destinations/commandbar/types.ts

```diff
@@ -0,0 +1,24 @@
+export type ModalFormFactor = { type: 'modal' }
+export type InlineFormFactor = { type: 'inline'; rootElement: string /* | HTMLElement */ }
```

Comment:
note we just fixed this [here](https://github.com/tryfoobar/monobar/pull/2250)

Reply:
Applied the new type also here now